src/index.html

```diff
@@ -5,9 +5,52 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="container">
+      <div class="container__card card" data-qa="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+          class="card__image"
+        >
+        <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star "></div>
+
+            <div class="stars__star "></div>
```

Comment:
```suggestion
            <div class="stars__star "></div>
            <div class="stars__star "></div>
```

No need to add blank line between block elements